Mathlib/Data/Nat/Find.lean

```diff
@@ -0,0 +1,92 @@
+/-
+Copyright (c) 2016 Microsoft Corporation. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Leonardo de Moura, Jeremy Avigad
+-/
+
+import Batteries.WF
+import Mathlib.Init.Data.Nat.Basic
+
+namespace Nat
```

Comment:
Can you move this after the module docstring, please? That will also silence the style linter.